src/definition/populate.go

```diff
@@ -14,9 +15,9 @@ type GuessFunc func(clusterName, groupLabel, entityID string, groups RawGroups)
 type PopulateFunc func(RawGroups, SpecGroups) (bool, []error)
 
 // MetricSetManipulator manipulates the MetricSet for a given entity and clusterName
-type MetricSetManipulator func(ms metric.MetricSet, entity sdk.Entity, clusterName string) error
+type MetricSetManipulator func(ms *metric.Set, entityMeta *integration.EntityMetadata, clusterName string) error
```

Comment:
Removing this will happen in a subsequent commit, right?

Reply:
It has not been removed in your tree. Will remove it :+1: